chrome/content/zotero/fileInterface.js

```diff
@@ -510,14 +510,17 @@ var Zotero_File_Interface = new function() {
 			format = "rtf";
 		}
 		
+		// determine locale preference
+		var locale = io.locale;
+		
 		// generate bibliography
 		try {
 			if(io.method == 'copy-to-clipboard') {
-				copyItemsToClipboard(items, io.style, false, io.mode === "citations");
+				copyItemsToClipboard(items, io.style, locale, false, io.mode === "citations");
```

Comment:
Do you know of any good documentation on public and private functions in JS?

When I use `this.copyItemsToClipboard(items, io.style, locale, false, io.mode === "citations");` I get "JavaScript error: chrome://zotero/content/fileInterface.js, line 519: TypeError: this.copyItemsToClipboard is not a function".